packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -356,6 +358,54 @@ class VifStatus extends BaseComponent {
   }
 }
 
+const AdvancedSettingsModal = decorate([
+  provideState({
+    effects: {
+      toggleTxChecksumming() {
+        const { onChange, value } = this.props
+        onChange({
+          ...value,
```

Comment:
Why do you need to keep it?

Reply:
The modal isn't specific to the TX checksumming, we shouldn't override other values.